### PR TITLE
Added client_id for non embedded signature requests

### PR DESCRIPTION
**Pull request by neillom**
Added client_id for non embedded requests to enable white-labeling.

Fix for reported issue #40

### CODE REVIEW

File: hellosign_sdk/hsclient.py [hunk 1]
```diff
@@ -1353,7 +1357,7 @@ def _check_required_fields(self, fields=None, either_fields=None):
                     raise HSException("One of the following fields is required: %s" % ", ".join(field.keys()))
 
     @api_resource(SignatureRequest)
-    def _send_signature_request(self, test_mode=False, client_id=None, files=None, file_urls=None, title=None, subject=None, message=None, signing_redirect_url=None, signers=None, cc_email_addresses=None, form_fields_per_document=None, use_text_tags=False, hide_text_tags=False, metadata=None, ux_version=None, allow_decline=False):
+    def _send_signature_request(self, test_mode=False, client_id=None, files=None, file_urls=None, title=None, subject=None, message=None, signing_redirect_url=None, signers=None, cc_email_addresses=None, form_fields_per_document=None, use_text_tags=False, hide_text_tags=False, metadata=None, ux_version=None, allow_decline=False, is_for_embedded_signing=None):
```
Comment (martinytodorov) [inline hellosign_sdk/hsclient.py]:
```suggestion
    def _send_signature_request(self, test_mode=False, client_id=None, files=None, file_urls=None, title=None, subject=None, message=None, signing_redirect_url=None, signers=None, cc_email_addresses=None, form_fields_per_document=None, use_text_tags=False, hide_text_tags=False, metadata=None, ux_version=None, allow_decline=False, is_for_embedded_signing=False):
```

**Review by martinytodorov — CHANGES_REQUESTED** [review]
Hey Marc, I added one suggestion and because I previously merged a big PR yours is now in conflict.

Can you try to fix it, or may be just pull the new changes that were merged and add your changes again (might be faster).

Thanks for adding that!

**Comment by neillom** [comment]
Hey Martin, included fix for issue I spotted after yesterday's merge.

**Review by martinytodorov — APPROVED** [review]
Looks good. This `.replace('null','{}')` will fix the signing_options for now, but I am about to open another PR that will move this into one of those format functions in HSFormat.

**Pull request merged**